src/openapi/peridio-admin-openapi.yaml

```diff
@@ -2379,7 +2379,7 @@ paths:
                   description: |
                     If omitted, `next_release_prn` will dictate where to create this release within the cohort's release graph.
 
-                    If supplied, `next_release_prn` is required to be supplied as well. Together, these fields allow the caller to insert a release between two other releases.
+                    In order to insert a release between two other releases, `next_release_prn` is required to be supplied as well.
```

Comment:
Can you also add a sentence that says if you supply neither field that it will create it as the latest automatically.

That sentence should be present in both next_release_prn and previous_release_prn